exe-struct.c

```diff
@@ -1,379 +0,0 @@
-#include <stdio.h>
```

Comment:
This file was moved to `tests/_exe/struct.c` in https://github.com/udem-dlteam/pnut/commit/47f1b8312eeb17434e67751e398b67829c417372#diff-a7c91b341afde4f0fedb076b6b357e48fa0cf625eca09826b611199eab02bb17